bitcoin_executor/sources/utxo.move

```diff
@@ -2,6 +2,12 @@
 
 module bitcoin_executor::utxo;
 
+use bitcoin_executor::utils::vector_slice;
+
+const OP_0: u8 = 0x00;
+const OP_DATA_20: u8 = 0x14;
+
+
 public struct OutPoint has copy, drop, store {
```

Comment:
please add comment to explain what is `OutPoint`

Reply:
done!